config/Dockerfiles/release-notes.py

```diff
@@ -129,5 +129,9 @@ def format_body(tasks):
 release_body = format_body(tasks)
 release_name = milestone
 
-repo.create_git_release(milestone, release_name, release_body,
-                        target_commitish="master")
+release = repo.create_git_release(milestone, release_name, release_body,
+                                  target_commitish="master")
```

Comment:
@14rcole we could release it to another test branch apart from master to mimic a mock release